Dockerfile.builder

```diff
@@ -31,5 +31,8 @@ ENV PATH=$PATH:/usr/local/go/bin
 RUN mkdir -p /tmp/go/bin && chmod a+rwx /tmp/go
 ENV GOPATH /tmp/go
 RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh && mv /tmp/go/bin/dep /usr/bin
-
+RUN cd $GOPATH/bin \
+	curl -L -s https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 -o dep \
```

Comment:
@xcoulon I bet that `${DEP_VERSION}` cannot be resolved because it comes from the make file.